exercises/variable-length-quantity/canonical-data.json

```diff
@@ -0,0 +1,185 @@
+{
+  "#": [
+    "JSON doesn't allow hexadecimal literals.",
```

Comment:
a shame. I suppose I could have them as strings (so, instead of `[255, 127]` the test data contains `["0xff", "0x7f"]`, but that seems a little wasteful.